rules/use-external-link.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const astUtils = require('jsx-ast-utils')
+const getElementType = require('eslint-plugin-jsx-a11y/lib/util/getElementType')
+
+module.exports =  {
+    meta: {
+        docs: {
+            description: 'Enforce all anchors with target="_blank" to use ExternalLink component',
+        },
+    },
+
+create: (context) => {
+    const elementType = getElementType(context);
+        return {
```

Comment:
Nit: The whitespace looks off, like `create` should be indented one leve, and the return block is one level too highly indented relative to `const elementType = getElementType(context);`